pkgs/development/python-modules/homeconnect/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, requests
+, requests_oauthlib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "homeconnect";
+  version = "0.6.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0n4h4mi23zw3v6fbkz17fa6kkl5v9bfmj0p57jvfzcfww511y9mn";
+  };
+
+  propagatedBuildInputs = [
+    requests
+    requests_oauthlib
+  ];
+
+  doCheck = true;
```

Comment:
```suggestion
```

Reply:
There are no tests for that package, so disabling `doCheck` is the way to go. Can we fetch the package from PyPi then?